### ISSUE TITLE
Multiple site views

### ISSUE THREAD

**Issue by j0hnsen** [nosmokingbandit/Watcher3#41]
If you got a huge list in watcher you need some time to load the whole list. It would help if there is a default view of 100 lines and multiple sites and maybe an option to view all lines of the list.

Thanks :)

**Comment by j0hnsen** [comment]
Any update here?

**Comment by nosmokingbandit** [comment]
In the big UI update a few things were changed to increase page loading time. I don't have a massive library, so you'll have to let me know if this made any noticeable difference.

All of the movies need to be sent to the browser so it can sort them according to title/year/status in ascending/descending order.

I ran some quick tests. An 5-fold increase in library size only added 15% to the loading time of the html, so I'm pretty sure most of the time is spent loading other elements -- like posters -- that are now lazy-loaded as needed.
